oggm/tests/test_models.py

```diff
@@ -350,6 +351,49 @@ def test_past_mb_model(self, hef_gdir):
                                                     years=mbdf.index.values)
         assert_allclose(s, mbdf['MY_MB'])
 
+    def test_prcp_fac_update(self, hef_gdir):
+
+        gdir = hef_gdir
+        init_present_time_glacier(gdir)
+
+        mb_mod = massbalance.PastMassBalance(gdir, bias=0,
+                                             check_calib_params=False)
+        # save old precipitation time series
+        prcp_old = mb_mod.prcp
```

Comment:
```suggestion
        prcp_old = mb_mod.prcp.copy()
```
Because of the optimisation above